inventory_management_system_api/schemas/catalogue_item.py

```diff
@@ -25,6 +25,7 @@ class PropertySchema(PropertyPostRequestSchema):
 
     name: str = Field(description="The name of the catalogue item property")
     unit: Optional[str] = Field(default=None, description="The unit of the property such as 'nm', 'mm', 'cm' etc")
+    unit_id: Optional[str] = Field(default=None, description="The ID of the unit")
```

Comment:
```suggestion
    unit_id: Optional[str] = Field(default=None, description="The ID of the unit")
    unit: Optional[str] = Field(default=None, description="The unit of the property such as 'nm', 'mm', 'cm' etc")
```
I would prefer unit_id to come first like the models.